_data/people/klieret.yml

```diff
@@ -0,0 +1,10 @@
+active: true
+focus-area:
+- ssc
+institution: Princeton University
+name: Kilian Lieret
+photo: /assets/images/team/Kilian-Lieret.jpg
+shortname: klieret
+title:
+website: https://lieret.net
+presentations:
```

Comment:
@klieret It is probably fine in another PR to backport some presentations that you've done for HSF training here if there was some cross over.